BusinessLogic/Models/Employee.cs

```diff
@@ -5,7 +5,7 @@ namespace BusinessLogic.Models;
 
 public class Employee : IModel
 {
-    public Guid Id { get; set; }
+    public Guid Id { get; set; } = Guid.NewGuid();
```

Comment:
ik zou dit gewoon doen bij het aanmaken, OF wel constructors gaan gebruiken. nu gaan gebruik we 2 manieren door elkaarheen gebruiken. (want waarom het id wel en andere gegevens niet)

Reply:
ok